docs/data-sources/clinvar.mdx

```diff
@@ -79,9 +79,9 @@ The Acc and Version fields are merged to form the ID (RCV000000001.2)
 ```
 We only use the field with Type="Preferred". Multiple phenotypes may be reported
 
-**Location and Variant Id**
+**Location, Variant Type and Variant Id**
```

Comment:
We should mention the variantTypes that are skipped:
1. Microsatellite
1. protein only
1. fusion
1. Complex
1. Variation
1. Translocation